_config.yml

```diff
@@ -25,7 +25,9 @@ pub-api: https://pub.dev/documentation
 pub-pkg: https://pub.dev/packages
 lints: https://dart-lang.github.io/linter/lints
 dartpad: https://dartpad.dev
-dartpadx: https://dartpad.dev/experimental/embed-new-dart.html
+dartpad-embed: https://dartpad.dev/experimental/embed-new-dart.html
+dartpad-embed-html: https://dartpad.dev/experimental/embed-new-html.html
+dartpad-embed-inline: https://dartpad.dev/experimental/embed-new-inline.html
```

Comment:
these should be:

- https://dartpad.dev/embed-dart.html
- https://dartpad.dev/embed-html.html
- https://dartpad.dev/embed-inline.html

the experimental/ directory will be removed at some point

Reply:
ptal